File: MedicineReminder-master/app/src/main/java/com/example/medicineremindernew/Pill.java
package com.example.medicineremindernew;

public class Pill {
    public int id;
    public String name;
    public int value;
    public String dosage;
    public String date1;
    public String date2;
    public String time;

    Pill(int id, String name, int value, String dosage, String date1, String date2, String time){
        this.id = id;
        this.name = name;
        this.value = value;
        this.dosage = dosage;
        this.date1 = date1;
        this.date2 = date2;
        this.time = time;
    }
}

File: app/src/main/java/com/example/medicineremindernew/models/Pill.kt
package com.example.medicineremindernew.models


import kotlinx.serialization.Serializable
import java.sql.Timestamp

@Serializable
data class Pill(
    var created_at: String? = null,
    var id: Long? = null,
    var user_id: String? = null,
    var name: String? = null,
    var dosage_value: Int? = null,
    var dosage_unit: String? = null,
    var date_from: String? = null,
    var date_to: String? = null,
    var times: List<String>? = null
): java.io.Serializable
